$NetBSD: patch-gettext-runtime_configure.ac,v 1.1 2015/04/11 10:44:37 bsiegert Exp $

Only run libasprintf configure when its build has been selected.
http://savannah.gnu.org/bugs/index.php?44808

--- gettext-runtime/configure.ac.orig	Fri Apr 10 19:47:35 2015
+++ gettext-runtime/configure.ac
@@ -172,7 +172,9 @@ if test "$CXX" = "no"; then
 fi
 AM_CONDITIONAL([ENABLE_LIBASPRINTF], [test -n "$SUBDIR_libasprintf"])
 
+if [ -n "$SUBDIR_libasprintf" ]; then
 AC_CONFIG_SUBDIRS([libasprintf])
+fi
 
 AC_CONFIG_FILES([Makefile])
 
